src/components/alert/SpshAlert.vue

```diff
@@ -70,5 +72,27 @@
 <style scoped>
   .v-alert {
     border-width: 3px;
+
+    @media screen and (max-width: 680px) {
+      display: flex;
+      flex-direction: column;
+    }
+  }
+
+  @media screen and (max-width: 680px) {
+    ::v-deep .v-alert__prepend {
+      align-self: center;
+      padding-bottom: 8px;
+    }
+
+    ::v-deep .v-alert__content {
+      align-self: unset;
+      text-align: center !important;
+    }
+
+    .v-alert__title {
+      display: flex !important;
+      justify-content: center;
+    }
   }
```

Comment:
Ich würde vielleicht das noch besser einstellen, in dem Ich das ganze in einer neuen Datei in "styles" packe mit dem Namen "alert.scss"

. ::v-depp wird das glaube Ich auch nicht mehr nötig sein, und das bleibt dann konform mit der Art und Weise, wie wir bisher Vuetify Components eingestellt haben.